FuelGaugeReader/FuelGaugeReader.ino

```diff
@@ -0,0 +1,38 @@
+#define USE_USBCON
+#include <ros.h>
+#include <std_msgs/Int8.h>
+std_msgs::Int8 percent_full;
+ros::Publisher pub_percent_full("percent_full", &percent_full);
+ros::NodeHandle nh;
+
+void setup()
+{
+  nh.initNode();
+  Serial.begin(9600);
+  nh.getHardware()->setBaud(9600);
+  nh.advertise(pub_percent_full);
+}
+
+long publisher_timer;
+const int READ_PIN = A7;
```

Comment:
Don't use alphanumeric values please. Check out this header file. 

What I've found is that using alphanumeric can lead to unnecessary obfuscation and it is too easy to have duplicate pin declarations. 

https://github.com/Greenzie/interface_board/blob/main/intrepid/p5/include/header.h

Reply:
I think I've got this fixed?